<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>329</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">mbGB-Qt</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Save state...</string>
     </property>
     <addaction name="actionSave1"/>
     <addaction name="actionSave2"/>
     <addaction name="actionSave3"/>
     <addaction name="actionSave4"/>
     <addaction name="actionSave5"/>
     <addaction name="actionSave6"/>
     <addaction name="actionSave7"/>
     <addaction name="actionSave8"/>
     <addaction name="actionSaveFile"/>
    </widget>
    <widget class="QMenu" name="menuLoad">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Load state...</string>
     </property>
     <addaction name="actionLoad1"/>
     <addaction name="actionLoad2"/>
     <addaction name="actionLoad3"/>
     <addaction name="actionLoad4"/>
     <addaction name="actionLoad5"/>
     <addaction name="actionLoad6"/>
     <addaction name="actionLoad7"/>
     <addaction name="actionLoad8"/>
     <addaction name="actionLoadFile"/>
    </widget>
    <addaction name="actionLoadROM"/>
    <addaction name="actionCloseROM"/>
    <addaction name="menuSave"/>
    <addaction name="menuLoad"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEmulation">
    <property name="title">
     <string>Emulation</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuSize">
     <property name="title">
      <string>Screen size</string>
     </property>
     <addaction name="action1x"/>
     <addaction name="action2x"/>
     <addaction name="action3x"/>
     <addaction name="action4x"/>
    </widget>
    <addaction name="actionGeneral"/>
    <addaction name="separator"/>
    <addaction name="menuSize"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEmulation"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <action name="actionLoadROM">
   <property name="text">
    <string>Open ROM...</string>
   </property>
  </action>
  <action name="actionCloseROM">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close ROM...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit...</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionGeneral">
   <property name="text">
    <string>General...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionSave1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionSave2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="actionSave3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actionSave4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="actionSave5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="actionSave6">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="actionSave7">
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="actionSave8">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>File...</string>
   </property>
  </action>
  <action name="action1x">
   <property name="text">
    <string>1x</string>
   </property>
  </action>
  <action name="action2x">
   <property name="text">
    <string>2x</string>
   </property>
  </action>
  <action name="action3x">
   <property name="text">
    <string>3x</string>
   </property>
  </action>
  <action name="action4x">
   <property name="text">
    <string>4x</string>
   </property>
  </action>
  <action name="actionLoad1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionLoad2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="actionLoad3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actionLoad4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="actionLoad5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="actionLoad6">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="actionLoad7">
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="actionLoad8">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="actionLoadFile">
   <property name="text">
    <string>File...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
